wfl/Database/MongoDBManager.cs

```diff
@@ -8,55 +8,31 @@
 
 namespace Symphono.Wfl.Database
 {
-    public class MongoDBManager: IDBManager
+    public class MongoDBManager<T>: IDBManager<T> where T: IEntity
     {
         private static string connectionString;
         public static MongoClient client;
         public static IMongoDatabase db;
+        private string collectionName { get; }
         
-        public MongoDBManager()
+        public MongoDBManager(string inputCollectionName)
         {
             connectionString = WebConfigurationManager.ConnectionStrings["databaseConnectionString"].ConnectionString;
```

Comment:
Since we are adding an injection constructor for dependency injection, let's move this connection string to a constructor parameter as well.